README.md

```diff
@@ -32,15 +34,28 @@ Here's how to use this:
 
 3. Add a stanza to your Home Assistant `configuration.yaml` like this:
 
-       climate:
-           - platform: smartthinq
-             refresh_token: YOUR_TOKEN_HERE
-             country: US
-             language: en-US
+       smartthinq:
+           token: [YOUR_TOKEN_HERE]
+           region: US [OPTIONAL]
+           language: en-US [OPTIONAL]
```

Comment:
I don't think we need the `[OPTIONAL]` markers. We can explain this in prose if necessary, but it's not really optional if your account is created in a non-US region.